runtime.txt

```diff
@@ -1 +0,0 @@
-3.7
```

Comment:
we need this for netlify.  it tells netlify to use python version 3.7.  without it, it defaults to python 2.7, which we are incompatible with

Reply:
i fixed